parm/fv3lam.xml

```diff
@@ -3404,5 +3410,11 @@
        <scale>3.0</scale>
        </param>
 
+       <param>
```

Comment:
@JesseMeng-NOAA Can you also update the xml file?

Reply:
Done